composer.json

```diff
@@ -5,14 +5,15 @@
     "description": "Blacklist plugin for Sylius.",
     "license": "MIT",
     "require": {
-        "php": "^8.0",
-        "sylius/sylius": "~1.11.0 || ~1.12.0"
+        "php": "^8.0 || ^8.1 || ^8.2 || ^8.3",
+        "sylius/sylius": "~1.12.0 || ~1.13.0",
+        "doctrine/annotations": "^2.0.0"
```

Comment:
`"doctrine/annotations": "^2.0.0"`
It seems to me that this is unnecessary but I could be wrong